### DISCUSSION TITLE
A lot of broken urls

### DISCUSSION THREAD

**Discussion by bobarros** [keystonejs/keystone#5589] [Questions] [answered]
Are you guys aware that a lot of links that supposedly will guide the user through installing process are leading to unexisting pages?

Like this one:
https://www.keystonejs.com/guide

**Comment by bobarros** [comment, marked as answer]
I'm sorry. I was following this tutorial below and ended up with a very old version installed.
Just did everything again and there are no broken links.

https://www.mongodb.com/blog/post/configuring-keystonejs-to-use-mongodb-atlas

**Comment by raveling** [comment]
Glad you figured it out! And sorry for the delayed reply. 😊
